client/app/host/QuestionAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+import { Switch } from '@/components/ui/switch';
+import {
+    Accordion,
+    AccordionContent,
+    AccordionItem,
+    AccordionTrigger,
+} from '@/components/ui/accordion';
+
+interface QuestionAccordionProps {
+    questions: {
+        question: string;
+        description: string;
+        language: string;
+        points: string;
+        tests: {
+            input: string;
+            output: string;
+        }[];
+        enabled: boolean;
+    }[];
+    handleQuestionSwitch: (question: string) => void;
+}
+
+const QuestionAccordion: React.FC<QuestionAccordionProps> = ({
+    questions,
+    handleQuestionSwitch,
+}) => {
+    return (
+        <Accordion type="single" collapsible className="w-full pl-2 pr-2">
+            {questions.map((q, index) => (
+                <AccordionItem
+                    key={index}
+                    value={`question-${index}`}
+                    className={`px-2.5} mb-1 rounded border`}
```

Comment:
Please be consistent about the use of ``className={`...`}`` vs `className="..."`

(Favour `className="..."`)